src/data/countryData.ts

```diff
@@ -15,606 +15,509 @@ import { CountryData } from '../types';
-      '269',
-      '270',
-      '276',
-      '281',
-      '301',
-      '302',
-      '303',
-      '304',
-      '305',
-      '307',
-      '308',
-      '309',
-      '310',
-      '312',
-      '313',
-      '314',
-      '315',
-      '316',
-      '317',
-      '318',
-      '319',
-      '320',
-      '321',
-      '323',
-      '325',
-      '330',
-      '334',
-      '336',
-      '337',
-      '352',
-      '360',
-      '361',
-      '386',
-      '401',
-      '402',
-      '404',
-      '405',
-      '406',
-      '407',
-      '408',
-      '409',
-      '410',
-      '412',
-      '413',
-      '414',
-      '415',
-      '417',
-      '419',
-      '423',
-      '425',
-      '432',
-      '434',
-      '435',
-      '440',
-      '478',
-      '479',
-      '480',
-      '501',
-      '502',
-      '503',
-      '504',
-      '505',
-      '507',
-      '508',
-      '509',
-      '510',
-      '512',
-      '513',
-      '515',
-      '516',
-      '517',
-      '518',
-      '520',
-      '530',
-      '540',
-      '541',
-      '559',
-      '561',
-      '562',
-      '563',
-      '570',
-      '573',
-      '574',
-      '575',
-      '580',
-      '585',
-      '586',
-      '601',
-      '602',
-      '603',
-      '605',
-      '606',
-      '607',
-      '608',
-      '609',
-      '610',
-      '612',
-      '614',
-      '615',
-      '616',
-      '617',
-      '618',
-      '619',
-      '620',
-      '623',
-      '626',
-      '630',
-      '631',
-      '636',
-      '641',
-      '650',
-      '651',
-      '660',
-      '661',
-      '662',
-      '701',
-      '702',
-      '703',
-      '704',
-      '706',
-      '707',
-      '708',
-      '712',
-      '713',
-      '714',
-      '715',
-      '716',
-      '717',
-      '718',
-      '719',
-      '724',
-      '727',
-      '731',
-      '732',
-      '734',
-      '740',
-      '757',
-      '760',
-      '763',
-      '765',
-      '770',
-      '772',
-      '773',
-      '775',
-      '781',
-      '785',
-      '801',
-      '802',
-      '803',
-      '804',
-      '805',
-      '806',
-      '808',
-      '810',
-      '812',
-      '813',
-      '814',
-      '815',
-      '816',
-      '817',
-      '818',
-      '828',
-      '830',
-      '831',
-      '843',
-      '845',
-      '847',
-      '850',
-      '856',
-      '858',
-      '859',
-      '860',
-      '863',
-      '864',
-      '865',
-      '870',
-      '901',
-      '903',
-      '904',
-      '906',
-      '907',
-      '908',
-      '909',
-      '910',
-      '912',
-      '913',
-      '914',
-      '915',
-      '916',
-      '918',
-      '919',
-      '920',
-      '925',
-      '928',
-      '931',
-      '936',
-      '937',
-      '940',
-      '941',
-      '949',
-      '951',
-      '952',
-      '954',
-      '956',
-      '970',
-      '972',
-      '973',
-      '978',
-      '979',
-      '985',
-      '989',
+      "201",
+      "206",
+      "210",
```

Comment:
Codes like 202, 203, 205, 207, 208, and 209 are missing now, but according to these resources they are valid:
https://www.bennetyee.org/ucsd-pages/area.html
https://www.allareacodes.com/area_code_listings_by_state.htm